wlauto/workloads/apklaunch/__init__.py

```diff
@@ -0,0 +1,77 @@
+#    Copyright 2015 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from time import sleep
+
+from wlauto import Workload, Parameter
+from wlauto import File
+from wlauto.exceptions import ConfigError
+
+
+class ApkLaunchWorkload(Workload):
+    name = 'apklaunch'
+    description = '''
+    Installs and runs a .apk file, waits wait_time_seconds, and tests if the app
+    has started successfully.
+    '''
+    supported_platforms = ['android']
+
+    parameters = [
+        Parameter('apk_file', description='Name to the .apk to run'),
+        Parameter('pkg_name', mandatory=True,
```

Comment:
I would change this to `package_name` or just `package`, for consistency with similar parameter namings elsewhere.

Also, this parameter isn't strictly necessary. You can obtain the package name from the APK with

    from wlauto.utils.android import ApkInfo
    self.package ApkInfo(self.apk_file).package